src/components/Navigation/Navigation.test.tsx

```diff
@@ -319,13 +328,27 @@ it("closes the mobile menu when opening the search form", () => {
 });
 
 it("can open the mobile menu", () => {
-  render(<Navigation logo={<img src="" alt="" />} />);
+  render(<Navigation items={items} logo={<img src="" alt="" />} />);
   const banner = screen.getByRole("banner");
   expect(banner.className.includes("has-menu-open")).toBe(false);
   userEvent.click(screen.getByRole("button", { name: "Menu" }));
   expect(banner.className.includes("has-menu-open")).toBe(true);
 });
 
+it("hides the mobile menu button when there are no navigation items to display", () => {
+  render(
+    <Navigation
+      items={undefined}
+      itemsRight={undefined}
```

Comment:
It looks like you can drop these props.

Reply:
I did this on purpose hoping it will help understand the relationship between specific props and output at a glance.